src/guides/v2.3/howdoi/checkout/checkout_order.md

```diff
@@ -110,3 +110,17 @@ In this file, add the following:
     </body>
 </page>
 ```
+
+## Step 4: Deploy the static content and Clean cache
```

Comment:
Hi @chandru-199. Thank you for your contribution. 
I believe this step should be more or less obvious for Magento developers. By copying over this information to all documents we might create an additional overhead

Reply:
Thanks @rogyar. But this is not same for all the documents right?. One process needs to clear cache and another one di: compile. Though It should be obvious, for new developers it will be definitely useful. Please share your thoughts.

Reply:
Hi @chandru-199. That makes sense. In that case we need to add a small clarification that you need to run `bin/magento setup:static-content:deploy` only if your Magento instance is in `production` mode.

Reply:
Thanks for your time @rogyar. Can we make it as notes that "Doesn't needed in developer mode"? in existing commit.

Reply:
Yes, please add this clarification

Reply:
Thanks @rogyar. I have added the clarification.I know we are contributing each and every thing to Magento for making it efficient one. And when focusing on devdocs, as a newbie I felt if these things  are present in these area, it will be more understandable and makes it complete tutorial.